src/view/renderer.js

```diff
@@ -116,6 +115,14 @@ export default class Renderer {
 		 * @type {null|HTMLElement}
 		 */
 		this._fakeSelectionContainer = null;
+
+		/**
+		 * Reference to the {@link module:engine/view/view~View#document}.
+		 *
+		 * @private
+		 * @member {module:engine/view/document~Document}
+		 */
+		this._viewDocument = view.document;
```

Comment:
Not needed anymore - this file needs to be reset to master state as we do not need to change anything here.